universe/interfaces.go

```diff
@@ -13,6 +13,14 @@ import (
 	"github.com/momentum-xyz/ubercontroller/utils/modify"
 )
 
+type Calendar interface {
+	Initialize(ctx context.Context, w World) error
+	Run() error
```

Comment:
cosmetic change: use `RunStopper` instead of the separate ones

Reply:
fixed